tests/integration/telemetry/stackdriver/testdata/custom_bootstrap.yaml.tmpl

```diff
@@ -12,5 +12,42 @@ data:
           "STACKDRIVER_MONITORING_EXPORT_INTERVAL_SECS": "10",
           "MESH_ID": "test-mesh",
         }
-      }
+      },
+      "tracing": {
+        "http": {
+          "name": "envoy.tracers.opencensus",
+          "typed_config": {
+            "@type": "type.googleapis.com/envoy.config.trace.v3.OpenCensusConfig",
+            "trace_config": {
+              "constant_sampler": {
+                "decision": "ALWAYS_PARENT"
+              },
+              "max_number_of_attributes": "200",
+              "max_number_of_annotations": "200",
+              "max_number_of_message_events": "200",
+              "max_number_of_links": "200"
+            },
+            "stackdriver_exporter_enabled": true,
+            "stackdriver_grpc_service": {
+              "google_grpc": {
+                "target_uri": "stackdriver.{{ .StackdriverNamespace }}.svc.cluster.local:8090",
```

Comment:
we can make this an option in envoy bootstrap itself? It should help with testing at other places too?

Reply:
are you suggesting we make the target URI configurable [here](https://github.com/istio/istio/blob/38e77b1a3587dcfce336c476529bb3bcf7501c80/tools/packaging/common/envoy_bootstrap.json#L596)?

That'd be something I'd be open to. I'm not sure whether or not we want to expose that bit of configurability though.

Reply:
@gargnupur how do you feel about revisiting that in a separate PR?